contracts/schemes/GenericSchemeMultiCallFactory.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity 0.5.17;
+
+import "./GenericSchemeMultiCall.sol";
+import "./SimpleSchemeConstraints.sol";
+
+/**
+ * @title 
+ */
+contract GenericSchemeMultiCallFactory {
+    uint8 public constant CUSTOM = 0;
+    uint8 public constant FAST = 1;
+    uint8 public constant NORMAL = 2;
+    uint8 public constant SLOW = 3;
+
+    function createGenericSchemeMultiCallSimple(
+        Avatar _avatar,
+        IntVoteInterface _votingMachine,
+        uint8 _voteParamsType,
+        bytes32 _voteParamsHash,
```

Comment:
lets add also the gov params..
something like https://github.com/daostack/arc/blob/master-2/contracts/schemes/ArcScheme.sol#L33